target/common/common.mk

```diff
@@ -9,7 +9,7 @@ DEBUG ?= OFF  # ON to turn on wave logging
 SIM_DIR  ?= $(shell pwd)
 TB_DIR   ?= $(SNITCH_ROOT)/target/common/test
 UTIL_DIR ?= $(SNITCH_ROOT)/util
-LOGS_DIR  = $(SIM_DIR)/logs
+LOGS_DIR ?= $(SIM_DIR)/logs
```

Comment:
Why? The framework now assumes that the `logs` directory lies at a specific relative path from the simulation directory. I'm afraid that allowing for different relative paths could break things. And if you don't need to allow for different relative paths then you can just rely on `SIM_DIR` being configurable.

Reply:
Addressed in [this ](https://github.com/pulp-platform/snitch_cluster/pull/137/commits/798b7b4c5801de6e049489d643729906081f3133)commit.